Ontology/Space/Components/SpaceOccupancy.json

```diff
@@ -25,9 +25,9 @@
     },
     {
       "@type": "Property",
-      "name": "inferredOccupancy",
+      "name": "hasInferredOccupancy",
```

Comment:
Naming makes sense, but I'm still very skeptical about the notion of the SpaceOccupancy component at all (see email thread). This really ought to be an InferredOccupancySensor.